src/meta/processors/admin/BalanceTask.cpp

```diff
@@ -61,10 +64,12 @@ void BalanceTask::invoke() {
             if (srcLived) {
                 client_->transLeader(spaceId_, partId_, src_).thenValue([this](auto&& resp) {
                     if (!resp.ok()) {
-                        LOG(ERROR) << taskIdStr_ << "Transfer leader failed, status " << resp;
                         if (resp == nebula::Status::PartNotFound()) {
-                            ret_ = BalanceTaskResult::INVALID;
+                            // if the partition has been removed before, regard as succeeded
+                            LOG(WARNING) << "Can't find part " << partId_ << " on " << src_;
```

Comment:
ERROR?

Reply:
It is not an error actually, it could happen when one part has been moved brfore.